apps/system/utils/README.md

```diff
@@ -183,6 +179,16 @@ PID | Pri |  Snap ticks  |
  * Snapshot interval : 10s (1000 ticks)
 --------------------------------------------------
 ```
+### How is the cpuload calculated
+We define a timeconstant (CONFIG_SCHED_CPULOAD_TIMECONSTANT), to keep a limit on total cpuload.
+We store "total ticks" and "ticks respective to a thread for specific core" after every systick interrupt.
+thread_n load on cpu_i = (thread_n load in cpu_i) / (total load in cpu_i)
```

Comment:
`thread_n load on cpu_i = (thread_n load in cpu_i) / (total load in cpu_i)` 
Change to 
`thread_n load on cpu_i = (thread_n ticks in cpu_i) / (total ticks in cpu_i)`